hack/update-codegen.sh

```diff
@@ -22,6 +22,22 @@ GO_CMD=${1:-go}
 KUEUE_ROOT=$(realpath $(dirname ${BASH_SOURCE[0]})/..)
 CODEGEN_PKG=$($GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 
+source "${CODEGEN_PKG}/kube_codegen.sh"
```

Comment:
This is already done below. Also the invocation of `kube::codegen::gen_helpers ` should be below `ln -s .. sigs.k8s.io`.

Reply:
Fixed